setup.py

```diff
@@ -67,6 +67,7 @@
             "dbt-duckdb>=1.7.1",
             "dbt-snowflake",
             "dbt-bigquery",
+            "dlt",
```

Comment:
I don't think we want this in 2 places. Let's just add the `dlt` target in Makefile where we need it.

Reply:
Sure, removed it from the `dev` requirements and added it the makefile in `install-dev` and `install-cicd-test` since `test_cli` is the test that requires it